src/patternfly/components/Masthead/masthead-content.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="pf-c-masthead__content{{#if masthead-content--modifier}} {{masthead-content--modifier}}{{/if}}"
+  {{#if masthead-content--attribute}}
+    {{{masthead-content--attribute}}}
+  {{/if}}>
+  {{#if @partial-block}}
+    {{> @partial-block}}
+  {{else}}
+    <span>Start content</span>
+    <span class="pf-u-ml-auto">End content</span>
```

Comment:
We probably shouldn't include utilities in the component examples, otherwise people might think it's part of the component. As long as this is just to show the size of the `__content` element, I think it's OK to remove it.